### PR TITLE
[WIP] update PyWPS documentation

### PR DESCRIPTION
**Pull request by tomkralidis**
Initial update of PyWPS documentation for review.

**Note**: I'll squash the PR once it's reviewed and fixes are made.

### CODE REVIEW

File: en/overview/pywps_overview.rst [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
+  :target: http://www.osgeo.org
 
 PyWPS
 ================================================================================
 
-Web Service
+Web Processing Service
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-PyWPS :doc:`(WPS) <../standards/wps_overview>` enables you to created and deploy
```
Comment (kalxas) [inline en/overview/pywps_overview.rst]: Please keep the link to the OGC standards documents

File: en/quickstart/pywps_quickstart.rst [hunk 2]
```diff
@@ -1,17 +1,17 @@
 :Author: OSGeo-Live
-:Author: Jachym Cepicky
-:Version: osgeo-live8.0-draft
-:License: Creative Commons Attribution-ShareAlike 3.0 Unported  (CC BY-SA 3.0)
+:Author: Jachym Cepicky, Tom Kralidis
+:Version: osgeo-live10.0
```
Comment (kalxas) [inline en/quickstart/pywps_quickstart.rst]: Has to be marked as -draft until someone does a full review from within the disk

File: en/overview/pywps_overview.rst (hunk 1 above)
Comment (tomkralidis) [inline en/overview/pywps_overview.rst]: Thanks, fixed.

File: en/quickstart/pywps_quickstart.rst (hunk 2 above)
Comment (tomkralidis) [inline en/quickstart/pywps_quickstart.rst]: Thanks, fixed.

**Comment by tomkralidis** [comment]
Closed in lieu of #128

**Pull request closed without merging**